client/src/views/Models/Model.vue

```diff
@@ -366,11 +374,19 @@
     }
 
     onNodeClick (node: GraphNodeInterface): void {
+      // Merge node metadata with Variables metadata. c.f. GraphNodeInterface type
+      const metadata = node.metadata ? node.metadata.flat() : null;
+
+      // Select which tab should be open first, then open the drilldown.
+      this.drilldownActiveTabId = metadata ? 'metadata' : 'parameters';
```

Comment:
All the metadata that comes from gromets should be displayed in the metadata tab. Then, the user could switch to see the parameters and knowledge tabs that should be populated with data coming from Cosmos. So I don't think this condition is really necessary?